packages/react/src/components/UIShell/HeaderGlobalAction.js

```diff
@@ -44,7 +44,7 @@ const HeaderGlobalAction = ({
       {...rest}
       {...accessibilityLabel}
       className={className}
-      onClick={onClick}
+      onClick={event => console.log(event)}
```

Comment:
Did you intend to replace this with something else?

Reply:
thanks no that was for testing I need to change it back.